package.json

```diff
@@ -10,10 +10,11 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "start": "nodemon server/index.js",
-    "seed-database": "mysql -u root < database/schema.sql"
+    "seed-database": "node database/faker.js && mysql -u root < database/schema.sql"
```

Comment:
Didn't know you could run one script to do two different things! -Does the script generate your database first then seeds it?

Reply:
yes! looked it up last night... you just write && then the other script you want to run. 
Right now I have it set up to run the faker.js file which adds the queryStrings into the sequel file, so they run in that order, but I really should refactor it, so that I create the database first, then just run the query string lines from the faker.js file.